tools/msvc/sbclang.targets

```diff
@@ -649,27 +667,26 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <Link Condition="'%(Link.ContainsStarboardLibraryDirectories)' != 'true'">
-        <AdditionalLibraryDirectories>%(Link.AdditionalLibraryDirectories);$(StarboardAllLibDirs)</AdditionalLibraryDirectories>
+      <Link Condition="'%(Link.AdjustedAdditionalOptions)' != 'true'">
         <AdditionalOptions Condition="$(AdditionalOptionsToUse) != ''">$(AdditionalOptionsToUse) %(Link.AdditionalOptions)</AdditionalOptions>
-        <ContainsStarboardLibraryDirectories>true</ContainsStarboardLibraryDirectories>
+        <AdjustedAdditionalOptions>true</AdjustedAdditionalOptions>
       </Link>
     </ItemGroup>
 
   </Target>
 
 
   <PropertyGroup>
-    <FixAdditionalLibraryDirectoriesLinkDependsOn>FixAdditionalLibraryDirectoriesWinMD;</FixAdditionalLibraryDirectoriesLinkDependsOn>
-    <FixAdditionalLibraryDirectoriesLinkDependsOn Condition="'$(WindowsAppContainer)' == 'true'">$(FixAdditionalLibraryDirectoriesLinkDependsOn);XamlPreLink</FixAdditionalLibraryDirectoriesLinkDependsOn>
+    <FixAdditionalOptionsLinkDependsOn>FixAdditionalOptionsWinMD;</FixAdditionalOptionsLinkDependsOn>
```

Comment:
Add $(FixAdditionalOptionsLinkDependsOn) to safeguard against squashing this property during future changes.